components/list/list-item-mixin.js

```diff
@@ -302,7 +302,6 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			:host([_focusing-primary-action]) [slot="outside-control-container"],
 			:host(:not([selection-disabled]):not([skeleton])[selected][_hovering-selection]) [slot="outside-control-container"],
 			:host(:not([selection-disabled]):not([skeleton])[selectable][_focusing]) [slot="outside-control-container"] {
-				background-color: white;
```

Comment:
Do you know if we have any visual-diff tests that capture cases where the page/parent background is non-white? I didn't dig into this, but I have a nagging concern that by removing the background, we could cause a regression for that case.

Reply:
I unfortunately don't know if we do - when I checked through a number of other tests that included list none of them seemed to contain anything similar. I share the worry of regression - I can spin up a sandbox to display what the behaviour may look like in order to reduce our concerns wrt regression

Reply:
I took a quick peek on [this demo page](https://live.d2l.dev/prs/BrightspaceUI/core/pr-4760/templates/primary-secondary/demo/integration.html), and it _seems_ ok.

Reply:
It would be nice if we had a select subset of visual-diff tests that covered that case. It requires a bit of thought though, to figure out which cases are "interesting". We probably don't want to simply double our tests. 

If we were to do that, I'd suggest adding those in a separate PR first, then updating this PR after that's been merged to see how this change affects them. Unless there is a rush to get this change in, that's probably the safest approach, which also helps protect us in the future too.

Reply:
List of tests I find interesting to have an offcolor background on:
- button-hover, button-focus
- all of the selectable tests
  - selectable - href
- draggable > color > selectable > hover
- separators - none

Reply:
We can expand on this but it's a decent starting point I think - I'll work on that other PR to add these tests